src/pudl/etl.py

```diff
@@ -330,7 +328,7 @@ def _load_static_tables_ferc1(datapkg_dir):
     return list(static_dfs.keys())
 
 
-def _etl_ferc1(etl_params, datapkg_dir, pudl_settings):
+def _etl_ferc1(etl_params, datapkg_dir, pudl_settings, unused_ds_kwargs):
```

Comment:
Due to the way `_etl_$x` functions are invoked, they all have to have the same signature, even though ferc1 does not need ds_kwargs (because it does not need to use datastore).

I've been used to mark arguments that are not required with `unused_` prefix, perhaps to make the lint checker happy but now that I am testing it it does not actually seem to be necessary.